From 660ae15f4af1f5ea8d9d50d4a24e4d91a94f9c2d Mon Sep 17 00:00:00 2001
From: Steve Peters <scpeters@openrobotics.org>
Date: Tue, 20 Feb 2024 17:25:23 -0800
Subject: [PATCH] Fix build with graphviz 10.0

Replace TRUE with 1, since the TRUE macro has been removed.

Signed-off-by: Steve Peters <scpeters@openrobotics.org>
---
 gazebo/gui/qgv/QGVSubGraph.cpp | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/gazebo/gui/qgv/QGVSubGraph.cpp b/gazebo/gui/qgv/QGVSubGraph.cpp
index 51925aba30..c16b69d065 100644
--- a/gazebo/gui/qgv/QGVSubGraph.cpp
+++ b/gazebo/gui/qgv/QGVSubGraph.cpp
@@ -44,13 +44,13 @@ QString QGVSubGraph::name() const
 
 QGVNode *QGVSubGraph::addNode(const QString &label)
 {
-    Agnode_t *node = agnode(_sgraph->graph(), NULL, TRUE);
+    Agnode_t *node = agnode(_sgraph->graph(), NULL, 1);
     if (node == NULL)
     {
         qWarning()<<"Invalid sub node :"<<label;
         return 0;
     }
-    agsubnode(_sgraph->graph(), node, TRUE);
+    agsubnode(_sgraph->graph(), node, 1);
 
     QGVNode *item = new QGVNode(new QGVNodePrivate(node), _scene);
     item->setLabel(label);
@@ -66,10 +66,10 @@ QGVSubGraph *QGVSubGraph::addSubGraph(const QString &_name, bool cluster)
     if (cluster)
     {
         sgraph = agsubg(_sgraph->graph(),
-            ("cluster_" + _name).toLocal8Bit().data(), TRUE);
+            ("cluster_" + _name).toLocal8Bit().data(), 1);
     }
     else
-        sgraph = agsubg(_sgraph->graph(), _name.toLocal8Bit().data(), TRUE);
+        sgraph = agsubg(_sgraph->graph(), _name.toLocal8Bit().data(), 1);
 
     if (sgraph == NULL)
     {
